openadapt/plotting.py

```diff
@@ -393,11 +395,11 @@ def plot_performance(
 
         ax.legend(all_handles, all_labels)
 
-    ax.set_title(f"{recording_timestamp=}")
+    ax.set_title(f"{recording.timestamp=}")
 
     # TODO: add PROC_WRITE_BY_EVENT_TYPE
     if save_file:
-        fname_parts = ["performance", str(recording_timestamp)]
+        fname_parts = ["performance", str(recording.timestamp)]
```

Comment:
This has been fixed in https://github.com/OpenAdaptAI/OpenAdapt/pull/774

Reply:
Yess @abrichr . i took the latest rebase and removed it